src/test/java/io/xstefank/wildfly/bot/InstallationEventTest.java

```diff
@@ -0,0 +1,42 @@
+package io.xstefank.wildfly.bot;
+
+import io.quarkiverse.githubapp.testing.GitHubAppTest;
+import io.quarkus.test.InMemoryLogHandler;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.QuarkusTestExtension;
+import org.jboss.logmanager.Level;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.kohsuke.github.GHEvent;
+
+import java.io.IOException;
+import java.util.logging.LogManager;
+
+import static io.quarkiverse.githubapp.testing.GitHubAppTesting.given;
+
+@QuarkusTest
+@GitHubAppTest
+public class InstallationEventTest {
+
+    private static final java.util.logging.Logger rootLogger = LogManager.getLogManager().getLogger("io.xstefank.wildfly.bot");
+    private static final InMemoryLogHandler inMemoryLogHandler = new InMemoryLogHandler(
+            record -> record.getLevel().intValue() >= Level.ALL.intValue());
+
+    static {
+        rootLogger.addHandler(inMemoryLogHandler);
+    }
+
+    @RegisterExtension
+    static QuarkusTestExtension TEST = new QuarkusTestExtension();
```

Comment:
why do you need this?

Reply:
You are right, there is no need for that. Removing that